ocs_ci/helpers/helpers.py

```diff
@@ -2972,3 +2973,72 @@ def run_cmd_verify_cli_output(
         if expected_output not in out:
             return False
     return True
+
+
+def check_pvs_present_for_ocs_expansion(sc=constants.LOCALSTORAGE_SC):
+    """
+    Check for pvs present for OCS cluster expansion
+
+    Args:
+        sc (str): Name of SC
+
+    Raises:
+        Assertion if PVS not present
+    """
+    flexible_scaling = config.ENV_DATA.get('enable_flexible_scaling')
+    arbiter_deployment = config.DEPLOYMENT.get("arbiter_deployment")
+
+    if not flexible_scaling and not arbiter_deployment:
+        for rack_no in range(0, 3):
+            pv_check_list = list()
+            nodes_obj = OCP(
+                kind=constants.NODE,
+                selector=f"{constants.TOPOLOGY_ROOK_LABEL}=rack{rack_no}",
+            )
+            nodes_data = nodes_obj.get()["items"]
+            node_name = [nodes["metadata"]["name"] for nodes in nodes_data]
```

Comment:
Here it will be more clear if we use the var name `node_names` instead of `node_name`.

Reply:
Done